src/protobuf/v1/publisher/publisher.proto

```diff
@@ -29,24 +34,24 @@ message SummaryResponse{
     int32 PodId = 9;
     string Operation = 10;
     string Labels = 11;
-    string DeploymentName = 12;
-    string Source = 13;
-    string Destination = 14;
-    string DestNamespace = 15;
-    string DestLabels = 16;
-    string NwType = 17;
-    string IP = 18;
-    int32 Port = 19;
-    string Protocol = 20;
-    string Action = 21;
-    int32 Count = 22;
-    int64 UpdatedTime = 23;
-    int32 WorkspaceId = 24;
-    string BindPort = 25;
-    string BindAddress = 26;
-    string Severity = 27;
-    string Tags = 28;
-    string Message = 29;
-    string Enforcer = 30;
-    string PolicyName = 31;
+    string Source = 12;
```

Comment:
In ProtoBuf, we should not re-assign the numbers to different fields. That will make the application incompatible with the older clients. We should always add new values at the end, incrementing last field number. 

To deprecate an existing field, either you can remove it from the definition or comment it out or mark the field as `deprecated=true`.
```proto
message MyMessage {
    // string my_field = 1
    string my_new_field = 2;
}

message MyMessage {
    string my_new_field = 2;
}

message MyMessage {
    string my_field = 1 [deprecated=true];
    string my_new_field = 2;
}
```
Marking it as `deprecated=true` seems to be a better choice.

Reply:
@AmanKumar2696 This is not not yet addressed properly. Please mark the old field as deprecated.

Reply:
For backward compatibility, this field wasn't marked as deprecated.

Reply:
Marking it as deprecated will not affect the backward compatibility.